debian/preinst

```diff
@@ -32,22 +32,22 @@ case "$1" in
     elif [ -x "/etc/init.d/${NAME}" ] || [ -e "/etc/init/${NAME}.conf" ]; then
       invoke-rc.d ${NAME} stop > /dev/null 2>&1 || true
     fi
-    # try and figure out if emby is running
-    PIDFILE=$(find /var/run/ -maxdepth 1 -mindepth 1  -name "emby*.pid" -print -quit)
-    [[ -n "$PIDFILE" ]] && [[ -s "$PIDFILE" ]] && EMBY_PID=$(cat ${PIDFILE})
+    # try and figure out if jellyfin is running
+    PIDFILE=$(find /var/run/ -maxdepth 1 -mindepth 1  -name "jellyfin*.pid" -print -quit)
+    [[ -n "$PIDFILE" ]] && [[ -s "$PIDFILE" ]] && JELLYFIN_PID=$(cat ${PIDFILE})
     # if its running, let's stop it
-    if [[ -n "$EMBY_PID" ]]; then
-      echo "Stopping Emby Server!"
-      # if emby is still running, kill it
-      if [[ -n "$(ps -p $EMBY_PID -o pid=)" ]]; then
-        CPIDS=$(pgrep -P $EMBY_PID)
+    if [[ -n "$JELLYFIN_PID" ]]; then
+      echo "Stopping Jellyfin!"
+      # if jellyfin is still running, kill it
+      if [[ -n "$(ps -p $JELLYFIN_PID -o pid=)" ]]; then
+        CPIDS=$(pgrep -P $JELLYFIN_PID)
         sleep 2 && kill -KILL $CPIDS
```

Comment:
This line looks reversed, shouldn't it be first "kill", then "sleep"? Sleep first is pointless IMHO

Reply:
These scripts were all copied direct from Carlos's build, so they need a bit more cleanup.